src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+Dokapi_url: http://localhost:9130
```

Comment:
I think we need to have possibility to override other properties https://github.com/folio-org/edge-common#system-properties available for user as VM or program arguments. Especially if we continue to use TokenCache.

Reply:
updated